modules/ROOT/pages/visualizer-app-network.adoc

```diff
@@ -52,22 +51,26 @@ https://en.wikipedia.org/wiki/Metric_prefix[Here is a quick review of metric pre
 
 == View Information About a Node in Your Network
 
-When you select a node within the graph, the node and its dependency stack are highlighted and all unconnected nodes are grayed out. Anypoint Visualizer displays information about the node in the panel on the right. This panel is divided into the following sections:
+When you select a node within the graph, the node and its dependency stack are highlighted and all unconnected nodes are grayed out. Anypoint Visualizer displays information about the node in a *Details/Monitoring* card. This card is divided into the following tabs:
```

Comment:
information about the node in a card

- same reasoning since the number of tabs might increase

Reply:
Accepted.